setup.py

```diff
@@ -36,22 +35,6 @@
 with open('requirements-dev.txt', encoding='utf-8') as f:
     tests_require = [str(req) for req in pkg_resources.parse_requirements(f)]
```

Comment:
Could/should we also remove this `tests_require`-related stuff (including below where we pass the `tests_require` parameter to the `setup()` function?

Reply:
I was thinking about that too and although it can be removed, I thought it might be better to do that when we are migrating the `pyproject.toml` approach, to reduce the risk of breaking something right now. What do you think?

Reply:
Sounds fine to me.